pkg/server/handler.go

```diff
@@ -0,0 +1,39 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package server
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+)
+
+type HandlerFunc func(c *gin.Context) error
+
+func WrapHandler(handler HandlerFunc) gin.HandlerFunc {
```

Comment:
WrapHandler -> ErrorHandler?

Reply:
We may not only deal with error here. The name comes from swag.WrapHandler.